README.md

```diff
@@ -33,3 +33,7 @@ Falcon is designed to help you manage your team's tasks and controls workflows i
 # The Team
 
 ![OurTeam](./static/img/OurTeam.jpeg)
+
+# Workflow Diagram
```

Comment:
Should be called DB diagram, it is not exactly workflow diagram

Reply:
At the SOW they are calling it ERD
https://github.com/redhat-beyond/documents/blob/main/Beyond06-SOW.md